### ISSUE TITLE
Source map support

### ISSUE THREAD

**Issue by ethul** [ethul/purs-loader#101]
Raising this on purs-loader

See ethul/purescript-webpack-example#4

**Comment by ethul** [comment]
Update: we may need to specifically return the sourcemap:

```javascript
this.callback(null, source, map);
```

**Comment by joprice** [comment]
Is there an example of using this feature? I tried adding sourceMaps to the pscArgs, based on the line here https://github.com/ethul/purs-loader/blob/59223097e35133918db227b3f75e009c1c46dc68/src/source-maps.js#L26, but I got an error that it's an invalid option:

```js
pscArgs: {
  sourceMaps: true
}
```

```
ERROR in Invalid option `--source-maps'

Usage: purs COMMAND
  The PureScript compiler and tools
```

Using `codegen: "js,sourcemaps"` seems to work to produce the map files in the output folder, but I can't get them to wind up in the dist/ folder when running webpack, or to work with the dev server. I tried using the source-map-loader as well, but the files don't seem to be recognized.
